### ISSUE TITLE
NUnit3TestAdapter 4.0.0: test owner no longer is written to .trx file

### ISSUE THREAD

**Issue by KlausDr** [nunit/nunit3-vs-adapter#878]
Since using version 4.0.0 of the NUnit3TestAdapter a test owner is no longer written to the .trx file
when using the TFS VSTest@2 task. (With NUnit3TestAdapter 3.17.0 this worked for me.)

I specify a test owner via the NUnit framework property 'Owner':
```
[assembly: NUnit.Framework.Property("Owner", "......")]
```

My context is:
- NUnit 3.12.0
- NUnit3TestAdapter 4.0.0
- Visual Studio 16.11.1
- TFS on-premises Dev18.M170.8 (using VSTest@2)

**Comment by OsirisTerje** [comment]
**Some notes from investigation**

Running with the Current discoverymode for the 4.0 adapter, gives the followinbg output from the DiscoveryUnderExecution part from NUnit.Engine/Framework:
```
<NUnitDiscoveryInExecution>d:\repos\NUnit\nunit3-vs-adapter.issues\Issue878\TestOwner\bin\x64\Debug\net48\TestOwner.dll</NUnitExecution>

<test-run id='0' name='TestOwner.dll' fullname='d:\repos\NUnit\nunit3-vs-adapter.issues\Issue878\TestOwner\bin\x64\Debug\net48\TestOwner.dll' runstate='Runnable' testcasecount='1'>
   <test-suite type='Assembly' id='0-1004' name='TestOwner.dll' fullname='d:\repos\NUnit\nunit3-vs-adapter.issues\Issue878\TestOwner\bin\x64\Debug\net48\TestOwner.dll' runstate='Runnable' testcasecount='1'>
      <test-suite type='TestSuite' id='0-1005' name='TestOwner' fullname='TestOwner' runstate='Runnable' testcasecount='1'>
         <test-suite type='TestFixture' id='0-1006' name='Tests' fullname='TestOwner.Tests' runstate='Runnable' testcasecount='1'>
            <test-case id='0-1001' name='Test1' fullname='TestOwner.Tests.Test1' methodname='Test1' classname='TestOwner.Tests' runstate='Runnable' seed='1174164608'>
               <properties />
            </test-case>
         </test-suite>
      </test-suite>
   </test-suite>
</test-run>

TFSFilter: TestCategory!=Foo
```
Using the legacy discovery mode, thus equal to 3.17, gives this result:
```
<test-run id='0' name='TestOwner.dll' fullname='d:\repos\NUnit\nunit3-vs-adapter.issues\Issue878\TestOwner\bin\x64\Debug\net48\TestOwner.dll' runstate='Runnable' testcasecount='1'>
   <test-suite type='Assembly' id='0-1002' name='TestOwner.dll' fullname='d:\repos\NUnit\nunit3-vs-adapter.issues\Issue878\TestOwner\bin\x64\Debug\net48\TestOwner.dll' runstate='Runnable' testcasecount='1'>
      <properties>
         <property name='Owner' value='Kermit the frog' />
         <property name='_PID' value='105268' />
         <property name='_APPDOMAIN' value='domain-14efe30d-TestOwner.dll' />
      </properties>
      <test-suite type='TestSuite' id='0-1003' name='TestOwner' fullname='TestOwner' runstate='Runnable' testcasecount='1'>
         <test-suite type='TestFixture' id='0-1000' name='Tests' fullname='TestOwner.Tests' classname='TestOwner.Tests' runstate='Runnable' testcasecount='1'>
            <test-case id='0-1001' name='Test1' fullname='TestOwner.Tests.Test1' methodname='Test1' classname='TestOwner.Tests' runstate='Runnable' seed='1174164608' />
         </test-suite>
      </test-suite>
   </test-suite>
</test-run>

TFSFilter: TestCategory!=Foo
```
That means we don't get any properties.  The discoverymode should not affect the engine, so have to look closer at what controls what here.

**Comment by OsirisTerje** [comment]
This issue has been fixed in NUnit 3.13
